client/src/components/MyBusinessList/MyBusinessList.tsx

```diff
@@ -45,32 +45,20 @@ export const MyBusinessList = () => {
         ) : (
           <>
             <div className='Profile-UserContainer_reviews business'>
-              <h4>your businesses</h4>
+              <h4>Your businesses</h4>
+              {/* temporary showing just business name because it was not showing anything */}
```

Comment:
Hi @sergiobtos!  May I suggest adding a `TODO` or `FIXME` tag at the beginning of your comment to highlight it better and assist in tracking the potential backlog functional and styling issues? 
 Thank you in advance for your consideration!

Reply:
Done